Source/EbsdLib/HKL/CtfReader.h

```diff
@@ -63,58 +63,39 @@ class EbsdLib_EXPORT CtfReader : public EbsdReader
-    EbsdHeader_INSTANCE_PROPERTY(CtfStringHeaderEntry, QString, Author, Ebsd::Ctf::Author)
-    EbsdHeader_INSTANCE_PROPERTY(CtfStringHeaderEntry, QString, JobMode, Ebsd::Ctf::JobMode)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, XCells, Ebsd::Ctf::XCells)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, YCells, Ebsd::Ctf::YCells)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, ZCells, Ebsd::Ctf::ZCells)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, XStep, Ebsd::Ctf::XStep)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, YStep, Ebsd::Ctf::YStep)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, ZStep, Ebsd::Ctf::ZStep)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, AcqE1, Ebsd::Ctf::AcqE1)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, AcqE2, Ebsd::Ctf::AcqE2)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, AcqE3, Ebsd::Ctf::AcqE3)
-    EbsdHeader_INSTANCE_PROPERTY(CtfStringHeaderEntry, QString, Euler, Ebsd::Ctf::Euler)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, Mag, Ebsd::Ctf::Mag)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, Coverage, Ebsd::Ctf::Coverage)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, Device, Ebsd::Ctf::Device)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, KV, Ebsd::Ctf::KV)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, TiltAngle, Ebsd::Ctf::TiltAngle)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<float>, float, TiltAxis, Ebsd::Ctf::TiltAxis)
-    EbsdHeader_INSTANCE_PROPERTY(CtfHeaderEntry<int>, int, NumPhases, Ebsd::Ctf::NumPhases)
-
-    EBSD_INSTANCE_PROPERTY(QVector<CtfPhase::Pointer>, PhaseVector)
-
-    EBSD_POINTER_PROP(Phase, Phase, int)
-    EBSD_POINTER_PROP(X, X, float)
-    EBSD_POINTER_PROP(Y, Y, float)
-    EBSD_POINTER_PROP(Z, Z, float)
-    EBSD_POINTER_PROP(BandCount, Bands, int)
-    EBSD_POINTER_PROP(Error, Error, int)
-    EBSD_POINTER_PROP(Euler1, Euler1, float)
-    EBSD_POINTER_PROP(Euler2, Euler2, float)
-    EBSD_POINTER_PROP(Euler3, Euler3, float)
-    EBSD_POINTER_PROP(MeanAngularDeviation, MAD, float)
-    EBSD_POINTER_PROP(BandContrast, BC, int)
-    EBSD_POINTER_PROP(BandSlope, BS, int)
-
-    /* These will be in a 3D ctf file */
-    EBSD_POINTER_PROP(GrainIndex, GrainIndex, int)
-    EBSD_POINTER_PROP(GrainRandomColourR, GrainRandomColourR, int)
-    EBSD_POINTER_PROP(GrainRandomColourG, GrainRandomColourG, int)
-    EBSD_POINTER_PROP(GrainRandomColourB, GrainRandomColourB, int)
-
-    /**
-     * @brief Returns the pointer to the data for a given feature
-     * @param featureName The name of the feature to return the pointer to.
-     */
-    void* getPointerByName(const QString& featureName);
+    EbsdHeader_INSTANCE_PROPERTY(CtfStringHeaderEntry, QString, Channel, Ebsd::Ctf::ChannelTextFile) EbsdHeader_INSTANCE_PROPERTY(CtfStringHeaderEntry, QString, Prj, Ebsd::Ctf::Prj)
```

Comment:
Macros can throw off indentation in IDEs unless there is a semicolon at the end.

Reply:
But some IDEs (QtCreator) throw a warning if they are missing.